ui/components/disclosure-form.js

```diff
@@ -0,0 +1,34 @@
+import { Transition, Disclosure } from '@headlessui/react'
```

Comment:
Nice work with this component. I would avoid making it it's own component / file until we need to (used by 2+)

Reply:
we are using in 2 places now .. is 2+ including 2? :P